docs/usage/ObjectStorage.md

```diff
@@ -204,5 +223,270 @@ aws s3api create-bucket \
      }
     ```
 
+---
+
+## GCP Object Storage
+
+This section covers setup of Object Storage with [*GCP Storage Buckets*](https://cloud.google.com/storage/docs/creating-buckets).
+
+### Create GCP Storage Bucket
+
+```bash
+# Set name of GCP Bucket
+BUCKET=<BUCKET_NAME_HERE>
+
+# Use the gsutil CLI to create the Storage Bucket
+gsutil mb gs://$BUCKET/
+```
+
+### Create a GCP Service Account for Velero to assume
+
+```bash
+# Set PROJECT_ID var to the currently active project
+PROJECT_ID=$(gcloud config get-value project)
+
+# Create a GCP Service Account
+gcloud iam service-accounts create velero \
+    --display-name "Velero Storage"
+
+# Set SERVICE_ACCOUNT_EMAIL to the email associated with the new Service Account
+SERVICE_ACCOUNT_EMAIL=$(gcloud iam service-accounts list \
+  --filter="displayName:Velero Storage" \
+  --format 'value(email)')
+
+
+# Grant necessary permissions to the new Service Account with the gsutil CLI
+ROLE_PERMISSIONS=(
+    compute.disks.get
+    compute.disks.create
+    compute.disks.createSnapshot
+    compute.snapshots.get
+    compute.snapshots.create
+    compute.snapshots.useReadOnly
+    compute.snapshots.delete
+    compute.zones.get
+)
+
+gcloud iam roles create velero.server \
+    --project $PROJECT_ID \
+    --title "Velero Server" \
+    --permissions "$(IFS=","; echo "${ROLE_PERMISSIONS[*]}")"    
+
+gcloud projects add-iam-policy-binding $PROJECT_ID \
+    --member serviceAccount:$SERVICE_ACCOUNT_EMAIL \
+    --role projects/$PROJECT_ID/roles/velero.server
+
+gsutil iam ch serviceAccount:$SERVICE_ACCOUNT_EMAIL:objectAdmin gs://${BUCKET}
```

Comment:
@eriknelson I'm pretty sure this line is responsible for granting admin privileges to the particular bucket we created. Makes sense that Velero needs access to _all_ disks, but only needs to be an Object Admin to this one particular bucket.